### PR TITLE
KJay/Homepage

### PR DESCRIPTION
**Pull request by KJay221**
1.finish bidInfo box
2.finish timeInfo box
3.finish RWD

### CODE REVIEW

**Comment by coliver0221** [comment]
Here are some comment:

## components/home/timeInfo.tsx
- Line.54
  - Please move this line to the top of the functional component with other `useState` variable
- Line.60~Line.76
  - You need to put all this codes in useEffect and listen to `currTime`'s changing.
  
## components/home/bidInfo.tsx
- Line.53~Line.59
  - You need to put all this codes in useEffect and listen to the `result`'s changing.
- Line.61~Line.71
  - Please make `listItem` a `useState` variable.
  
## components/home/listContent.tsx
- Line.26~Line.29
  - I think it's better to user conditional operator here.
- Line.32~Line.34
  - Please make `sellState` an `useState` variable.

**Comment by KJay221** [comment]
## components/home/timeInfo.tsx
- Line.54
  - Done
- Line.60~Line.76
  - Done
  
## components/home/bidInfo.tsx
- Line.53~Line.59
  - Done
- Line.61~Line.71
  - Because it is more easy to rerender component and listening time to change listItem,I create a new component and move something into it.
  
## components/home/listContent.tsx
- Line.26~Line.29
  - If change it will cause some lint problem so I didn't change it.
- Line.32~Line.34
  - Done

File: src/components/home/timeInfo.tsx [hunk 1]
```diff
@@ -45,38 +51,40 @@ const TimeInfo: React.FC = () => {
...
+      if (result[i].status === '執行中') {
+        if (result[i].bid_type === 'buy') {
+          setBuyPrice(String(result[i].wins.price));
+          setBuyValue(String(result[i].wins.value));
+        } else if (result[i].bid_type === 'sell') {
+          setSellPrice(String(result[i].wins.price));
+          setSellValue(String(result[i].wins.value));
+        }
       }
     }
```
Comment (coliver0221) [inline src/components/home/timeInfo.tsx:72]: You should put line.64~line.74 in `useEffect` and listen to the change of `result`.

File: src/components/home/timeInfo.tsx [hunk 2]
```diff
@@ -45,38 +51,40 @@ const TimeInfo: React.FC = () => {
...
+  }, []);
+
+  useEffect(() => {
+    update();
+  }, [currTime]);
 
   useEffect(() => {
-    setInterval(getCurrTime, 3000);
+    setInterval(getCurrTime, 60000);
   }, []);
```
Comment (coliver0221) [inline src/components/home/timeInfo.tsx]: You can put two `useEffect` together if they both listen to `[]`.

**Review by coliver0221 — CHANGES_REQUESTED** [review]

**Comment by coliver0221** [comment]
Please add the `box-shadow` to `time info block` and `bidding info block`.

File: src/components/home/timeInfo.tsx (hunk 2 above)
Comment (KJay221) [reply src/components/home/timeInfo.tsx]: done

File: src/components/home/timeInfo.tsx (hunk 1 above)
Comment (KJay221) [reply src/components/home/timeInfo.tsx:72]: done

**Comment by coliver0221** [comment]
LGTM.

**Pull request merged**